viewer/src/migration/NodeIdAndTreeIndexMaps.ts

```diff
@@ -5,15 +5,20 @@
 import { Subject, Observable } from 'rxjs';
 import { bufferTime, flatMap, filter, mergeAll, map, share, tap, first } from 'rxjs/operators';
 import { CogniteClient, InternalId, Node3D } from '@cognite/sdk';
+import { ParsedSector } from '../data/model/ParsedSector';
+import { LevelOfDetail } from '../data/model/LevelOfDetail';
+import { Sector, SectorQuads } from '../models/cad/types';
 
 type NodeIdRequest = InternalId;
 
 export class NodeIdAndTreeIndexMaps {
   readonly nodeIdToTreeIndexMap: Map<number, number>;
+  readonly treeIndexToNodeIdMap: Map<number, number>;
   readonly nodeIdRequestObservable: Subject<NodeIdRequest>;
   readonly nodeIdResponse: Observable<Node3D>;
   constructor(modelId: number, revisionId: number, client: CogniteClient) {
     this.nodeIdToTreeIndexMap = new Map();
+    this.treeIndexToNodeIdMap = new Map();
     this.nodeIdRequestObservable = new Subject();
     this.nodeIdResponse = this.nodeIdRequestObservable.pipe(
       bufferTime(50),
```

Comment:
Is there any reason to wait? Or will risk choppy performance from this? I guess 50 is quite low though. Just want to make sure.

Reply:
Is this comment misplaced? There is no waiting here.

Reply:
This is from the other PR for requesting node IDs from the API if they are not available locally. The wait time here is added in cases there are thousands of simultaneous requests. If there is no waiting, we will be sending one API request for each missing node ID, which I think we agreed that we wanted to avoid. However, there is no waiting if the data is already available after the current PR, because then you will already have the data available from the files. There is only a wait in case you are trying to set the color or hide a node that we currently do not know the tree index for.